docs/source/release_notes/release-notes.rst

```diff
@@ -44,6 +44,22 @@ develop
     *    - Type
          - Change
 
+    *    -
+         -
+
+=======
+v0.52.0
+=======
+
+1 August 2017
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Type
+         - Change
+
```

Comment:
Could you also move the devbreak to after the fixes, please? It's much less important (2224 >> 2210 >> 1933)